workflow-templates/markdown_links_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ignorePatterns": [
+      {
+        "pattern": "^https://github.com/cheqd/cheqd-tooling"
+      },
+      {
+        "pattern": "^https://docs.starport.network/intro/install.html"
+      }
+    ]
+}
```

Comment:
This is one solution, but the problem with it is that it will only work for specific links that need to be maintained manually - which will be a real pain over time.

Instead, I would like to use one of the other solutions:
1. Provide access to the private repo using a token during the execution. See the last section on this link: https://www.pinel.cc/blog/2019/11/02/private-github-actions
2. If it's NOT possible to do the above, then if we want to skip private repo checks the more sustainable way to do it is to skip checks for any links that return a 404 error for repositories. This is not an ideal way to do it, since it can result in certain repos not being checked. @askolesov found a way to do this by defining the HTTP status codes in mlc_config.json here: https://github.com/cheqd/cheqd-node/blob/main/mlc_config.json

Reply:
I used the GH_PAT variable to could access and check the private links.
If is a link that is from outside the repo we will need to exclude it becuase this only get the access to the current repo and its private links.